perception_ecu_launch/launch/perception_ecu.launch.py

```diff
@@ -0,0 +1,188 @@
+
+from launch import LaunchDescription, LaunchContext
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription, OpaqueFunction, LogInfo
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration, PathJoinSubstitution
+import yaml
+
+
+def create_launch_arg(name: str, default_value=None):
+    return DeclareLaunchArgument(name, default_value=default_value)
+
+
+def create_include_launch(path: list, args: dict, cond=IfCondition("True")):
+    return IncludeLaunchDescription(PythonLaunchDescriptionSource(path), launch_arguments=args.items(), condition=cond)
+
+
+def create_node(pkg: str, exec: str, ns: str, remaps=None, params=None, cond=IfCondition("True")):
+    return Node(package=pkg, executable=exec, namespace=ns, output="screen", remappings=remaps, parameters=params, condition=cond)
+
+
+def generate_arguments():
+
+    actions = []
+
+    actions.append(create_launch_arg("camera_modes_path", PathJoinSubstitution(
+        [FindPackageShare("perception_ecu_launch"), "config/camera_modes.yaml"])))
+
+    actions.append(create_launch_arg("sensing_namespace", "/sensing/camera"))
+    actions.append(create_launch_arg("image_topic_name", "image_raw"))
+    actions.append(create_launch_arg("vehicle_id", "default"))
+    actions.append(create_launch_arg("use_sensor_data_qos", "False"))
+
+    actions.append(create_launch_arg("launch_sensor_trigger", "True"))
+
+    actions.append(create_launch_arg("perception_namespace",
+                   "/perception/object_recognition/detection"))
+    yolox_default_basedir = [FindPackageShare("tensorrt_yolox"), "data"]
+    actions.append(create_launch_arg("yolox_model_path", PathJoinSubstitution(
+        yolox_default_basedir + ["yolox-tiny.onnx"])))
+    actions.append(create_launch_arg("yolox_label_path",
+                   PathJoinSubstitution(yolox_default_basedir + ["label.txt"])))
+
+    actions.append(create_launch_arg("launch_image_view", "True"))
+
+    return actions
+
+
+def generate_camera_processes(context: LaunchContext):
+
+    actions = []
+
+    camera_modes_path = context.launch_configurations["camera_modes_path"]
+    enable_trigger = bool(context.launch_configurations["launch_sensor_trigger"])
+    enable_viewer = bool(context.launch_configurations["launch_image_view"])
+
+    try:
+        with open(camera_modes_path, "r") as f:
+            camera_modes = yaml.safe_load(f)
+    except Exception as e:
+        print(e)
+        return
+
+    def create_indivisual_param_path(filename):
+        config_basedir_path = PathJoinSubstitution(
+            [FindPackageShare("individual_params"), "config", LaunchConfiguration("vehicle_id")])
+        return PathJoinSubstitution([config_basedir_path, camera_name, filename])
+
+    def create_launch_path(pkgname, filename):
```

Comment:
```suggestion
    def create_launch_path(pkg_name, filename):
```
nits

Reply:
`filename` exists as a noun in a dictionary. I think `file_name` is also suitable to unify the style.

Reply:
I'll leave it to you.

Reply:
good point!
https://github.com/tier4/perception_ecu_launch/pull/3/commits/c6b8f65a0aab96bc1f167464bb34c580973b5a9e